src/components/pages/ParentHome/ParentCalendar.js

```diff
@@ -1,61 +1,185 @@
+          instructor_rating,
+        } = item;
+        return (
+          <li className="cell" key={session_id}>
+            <Badge className="badge capital" status={type} text={subject} />
+            <Button
+              onClick={() => showModal(session_id)}
+              type="primary"
+              shape="round"
+              icon={<InfoCircleOutlined />}
+              size={'small'}
+            >
+              {' '}
+              details
+            </Button>
+            {session_id === activeModal ? (
+              <Modal
+                className="events capital"
+                title={subject}
+                visible={true}
+                onCancel={handleCancel}
+                footer={[
+                  <Button key="cancel" onClick={handleCancel}>
+                    Cancel
+                  </Button>,
+                  <Button key="book" type="primary" onClick={handleClick}>
+                    Book Now!
+                  </Button>,
+                ]}
+              >
+                <div className="capital">course description: {description}</div>
+                <div>
+                  prerequisites:
+                  {prereqs.length > 0 ? (
+                    prereqs.map((itm, idx) => {
+                      return <p key={idx}>{itm}</p>;
+                    })
+                  ) : (
+                    <>N/A</>
+                  )}
+                </div>
+                <div>first day of class: {dateConverter(start_date)}</div>
+                <div>last day of class: {dateConverter(end_date)}</div>
+                <div className="time">
+                  time: {timeConverter(start_time)} - {timeConverter(end_time)}
+                </div>
+                <div>
+                  location:<a href={location}>{location}</a>
+                </div>
+                <div>instructor: {instructor_name}</div>
+                <div>instructor rating: {instructor_rating}</div>
+              </Modal>
+            ) : null}
+          </li>
+        );
+      })}
+    </ul>
+  );
+}
 
-  const [isModalVisible, setIsModalVisible] = useState(false);
+function ParentCalendar() {
+  const [courses, setCourses] = useState(dummyAvailableCourses);
   const [activeModal, setActiveModal] = useState(0);
-  const [fullProgram, setFullProgram] = useState(false);
 
-  function getListData(value) {
-    schedule.courses.map(items => {
-      setCourse(items);
-    });
+  // when the calendar mounts, get the available courses/sessions from reducer and set it to courses
+
+  const today = new Date().toISOString().slice(0, 10);
+  const thisMonth = new Date().toISOString().slice(0, 7);
 
+  function getListData(value) {
     let listData = [];
-    let today = new Date().toLocaleString('en-US', {
-      day: '2-digit', // numeric, 2-digit
-      year: 'numeric', // numeric, 2-digit
-      month: '2-digit', // numeric, 2-digit, long, short, narrow
-    });
-    // console.log(schedule.sessions);
```

Comment:
Missed one! Please get ride of this console log

Reply:
this is the one I deleted. the changed code is on the right hand side though